modules/kubernetes/ingress-nginx/templates/values.yaml.tpl

```diff
@@ -35,6 +35,11 @@ controller:
     ${key}: "${value}"
     %{~ endfor ~}
     server-tokens: "false"
+    use-forwarded-headers: "true"
```

Comment:
Is this something that we always wants to do?
Could any of our users be relying of this value being "false"?

Reply:
We should always have X-Forwarded-For in my opinion.